bitnami/mysql/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: mysql
 sources:
   - https://github.com/bitnami/bitnami-docker-mysql
   - https://mysql.com
-version: 8.4.4
+version: 8.4.5
```

Comment:
Please bump the minor version since this is a new feature

Reply:
Done, I assumed as feature was already there it was not working thats why updated patch version .

Reply:
Thank you for changing it. The feature was not there, in this case the docs were incorrect so the feature is actually new.